### DISCUSSION TITLE
error install library in R Studio

### DISCUSSION THREAD

**Discussion by hanesegi** [teamalgoritma/community#449] [Q&A] [answered]
Try removing 'C:\Users\User\AppData\Local\R\win-library\4.2/00LOCK' Calls: sourceWithProgress ... eval -> <Anonymous> -> .install.winbinary -> unpackPkgZip

solusinya bagaimana ya kak?
terima kasih

**Comment by divakartika** [comment]
Dear @hanesegi 

Boleh dikonfirmasi library apa yang ingin diinstall? Dapat dicantumkan juga screenshot atau copy code error secara lengkap?

**Comment by kevbow** [comment, marked as answer]
Halo @hanesegi, untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini karena belum ada respon setelah 7 hari respon terakhir. Semoga dapat dipahami.
